packages/atomic/src/components/common/search-box/search-text-area.tsx

```diff
@@ -20,7 +20,7 @@ interface Props extends JSXBase.TextareaHTMLAttributes<HTMLTextAreaElement> {
 
 function getPopupAttributes(props: Required<Props>['popup']) {
   return {
-    role: 'combobox',
```

Comment:
`combobox` is not a valid role for a `textarea` element

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#technical_summary

Reply:
Using a `textarea` element instead of an `input` could be considered an accessibility issue because it means we can't fully implement the recommended combobox pattern.

In particular:

> The element that serves as an input and displays the combobox value has role [combobox](https://w3c.github.io/aria/#combobox).

See:

- https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/grid-combo/
- https://www.w3.org/WAI/ARIA/apg/patterns/combobox/#roles_states_properties

However, in the wild we can see that both Bing and Google use textarea with combox. In the case of Google, they specify `combobox` as a `role` (which is plain wrong and raises an accessibility issue when running a Lighthouse scan), whereas Bing uses the implicit `textbox` role, which flags no accessibility issues.

I believe the best solution is to eliminate the role (as Bing does).